plugins/github-issues/src/api/gitHubIssuesApi.ts

```diff
@@ -151,20 +189,50 @@ export const gitHubIssuesApi = (
   return { fetchIssuesByRepoFromGitHub };
 };
 
+function formatFilterValue(value: IssuesFilters[keyof IssuesFilters]): string {
+  if (Array.isArray(value)) {
+    return `[ ${value.map(formatFilterValue).join(', ')}]`;
+  }
+
+  return typeof value === 'string' ? `\"${value}\"` : `${value}`;
+}
+
+function createFilterByClause(filterBy?: IssuesFilters): string {
+  if (typeof filterBy === 'undefined') {
+    return '';
+  }
+
+  return Object.entries(filterBy)
+    .flatMap(([field, value]) => {
+      if (typeof value === 'undefined') {
```

Comment:
```suggestion
      if (value) {
```

Reply:
@jhaals should that not be `if (!value) {` , i.e. not truthy?

Reply:
yes you're right :)